src/api-helper.ts

```diff
@@ -9,6 +10,8 @@ export function errToStr(error: unknown, def: string) {
     return error.response.data.message
   } else if (isApiErrorResponse(error)) {
     return error.message
+  } else if (error instanceof ErrorEvent) {
+    return error.code ? `${error.code}: ${error.message}` : error.message?.toString() || def
```

Comment:
Since error.message has type `string | undefined`, it seems like we should do `return error.code ? `${error.code}: ${error.message || def}` : error.message || def` here. That gives us a fallback message in all cases and avoids the unnecessary `toString` call on something we already know is a string.

Reply:
Made this change, thanks